engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -39,7 +41,7 @@ class CromwellApiServiceActor(val workflowManager: ActorRef, config: Config)
   def receive = runRoute(possibleRoutes)
 }
 
-trait CromwellApiService extends HttpService with PerRequestCreator {
+trait CromwellApiService extends HttpService with PerRequestCreator with ServiceRegistry {
```

Comment:
Wait... is this right? I don't think the REST API should also be the service registry!?